crates/biome_js_analyze/src/lint/suspicious/no_comment_text.rs

```diff
@@ -77,29 +112,29 @@ impl Rule for NoCommentText {
     fn action(ctx: &RuleContext<Self>, _: &Self::State) -> Option<JsRuleAction> {
         let node = ctx.query();
         let mut mutation = ctx.root().begin();
+        let node_text = node.text().to_string();
+
+        // Replace the comments with JSX comments
+        let new_jsx_value = COMMENT_REGEX.replace_all(&node_text, |caps: &regex::Captures| {
```

Comment:
This is just my preference (not a suggestion) for readability. What do you think?

```
let new_jsx_value = COMMENT_REGEX.replace_all(&node_text, |caps: &regex::Captures| {
    let comment = caps[0]
        .trim_start_matches("//")
        .trim_start_matches("/*")
        .trim_end_matches("*/")
        .trim();
    format!("{{/*{}*/}}", comment)
});
```

Reply:
@chansuke 
To accommodate Doc comments(like `/** example */`), the code has been changed as follows

```rs
let new_jsx_value = COMMENT_REGEX.replace_all(&node_text, |caps: &regex::Captures| {
    let comment = caps[0].trim();
    match comment {
        c if c.starts_with("//") => format!("{{/* {} */}}", c[2..].trim()),
        c if c.starts_with("/**") => format!("{{/** {} */}}", &c[3..c.len() - 2].trim()),
        c => format!("{{/* {} */}}", &c[2..c.len() - 2].trim()),
    }
});
```

If you have any suggestions for a better code, I welcome them!